eZ/Publish/Core/Base/ServiceContainer.php

```diff
@@ -225,7 +225,7 @@ protected function dumpContainer(ConfigCache $cache)
     protected function prepareDirectory($directory, $name)
     {
         if (!is_dir($directory)) {
-            if (false === @mkdir($directory, 0777, true)) {
+            if (!@mkdir($directory, 0777, true) && !is_dir($directory)) {
```

Comment:
Could use an explanation here, please.

Reply:
Sure thing: https://github.com/kalessil/phpinspectionsea/blob/master/docs/probable-bugs.md#mkdir-race-condition =) Concurrency is fun, isn't it?